cfr.yaml

```diff
@@ -1291,3 +1291,39 @@
     lifetime: 3
   targeting: firefoxVersion >= 74 && localeLanguageCode == 'en' &&
     userPrefs.cfrFeatures && region == 'US'
+- id: DOH_ROLLOUT_CONFIRMATION
+  targeting: "'doh-rollout.enabled'|preferenceValue && !'doh-rollout.disable-heuristics'|preferenceValue && !'doh-rollout.skipHeuristicsCheck'|preferenceValue && !'doh-rollout.doorhanger-decision'|preferenceValue && firefoxVersion >= 79"
```

Comment:
I can't confirm the DoH related prefs in this targeting to my knowledge, do we have a reference for those prefs?

Reply:
Code: https://firefox-source-docs.mozilla.org/toolkit/components/messaging-system/docs/TriggerActionSchemas/TriggerActionSchemas.html
Doc discussion: https://docs.google.com/document/d/184dUI1bMauDWkxcm94tKouyiOWNZ7QGhfnOqYTRf3P4/edit#heading=h.5840t7jz7l1f

Reply:
Thanks! That's very helpful.
🚢